### PR TITLE
Added Symfony3 compatibility and fix tests

### PR DESCRIPTION
**Pull request by draco1989**

### CODE REVIEW

File: Form/Type/EnumType.php [hunk 1]
```diff
@@ -63,23 +61,26 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
...
-        $resolver->setDefaults(array(
-            'enum_class' => $enumClass,
-            'choices'    => $choices,
-        ));
-
-        $resolver->setAllowedTypes(array(
-            'enum_class' => 'string'
-        ));
+        $resolver
+            ->setDefined(array('enum_class', 'choices'))
```
Comment (yethee) [inline Form/Type/EnumType.php]: There is no need to define the options when we set default values.

File: Form/Type/EnumType.php [hunk 2]
```diff
@@ -63,23 +61,26 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
...
+                array(
+                    'enum_class' => $enumClass,
+                    'choices' => $choices,
+                )
+            )
+            ->setAllowedTypes('enum_class', array('string'))
+            ->setAllowedTypes('choices', array('array'));
     }
 
+
```
Comment (yethee) [inline Form/Type/EnumType.php]: Can you remove extra line?

File: composer.json [hunk 3]
```diff
@@ -9,14 +9,15 @@
     ],
 
     "require": {
-        "php": ">=5.3.3",
-        "symfony/framework-bundle": "~2.1",
-        "symfony/form": "~2.1"
+        "php": ">=5.5.9",
+        "symfony/framework-bundle": "~2.7",
+        "symfony/form": "~2.7"
```
Comment (yethee) [inline composer.json]: This constraint of version is not allowing `3.0`. We should use `~2.7|~3.0` constraint.

**Comment by yethee** [comment]
@draco1989 Thank for PR.

File: Form/Type/EnumType.php (hunk 1 above)
Comment (draco1989) [inline Form/Type/EnumType.php]: With new api, seems to be mandatory, you can see Travis. My first commits didn't have it, I had to add this.

Comment (yethee) [inline Form/Type/EnumType.php]: Strange bug. Tests were run for old (2.1-2.3) versions of Symfony.
https://travis-ci.org/yethee/BiplaneEnumBundle/builds/95997265

**Comment by yethee** [comment]
Can you replace `Symfony\Component\Form\Tests\FormIntegrationTestCase` to `Symfony\Component\Form\Test\FormIntegrationTestCase`? And I'll merge this PR.

**Comment by draco1989** [comment]
Now it's ported. I had to flip default choices due to Symfony3 finally removed the option "choices_as_values". Also I added as default option in case of Symfony >=2.7 && <3.0

**Comment by yethee** [comment]
[2.0.0](https://packagist.org/packages/yethee/enum-bundle#v2.0.0) is released. 
Thank you for the patch.

**Pull request merged**